content/english/python-interactive/activity-3.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Activity 3 - Welcome Customer to your Restaurant & Display Menu"
+date: 2019-07-25T13:24:17-07:00
+weight: 3
+draft: false
+translationKey: "python-INTERACTIVE-activity-3"
+---
+## Welcome Customer to your Restaurant & Display Menu
+Now that we have both a restaurant banner to display to the customer and a menu stored in your Python Program, let's display your menu to the customer and welcome them in. In this activity, we're going to write 2 helper functions to accomplish this.
+{{% notice %}}
+## Helper Functions
+Helper functions are functions that get called in a main function that runs all of the code for this program. Each of these helper functions works toward one simple goal. 
+{{% /notice tip%}}
+
```

Comment:
Consider adding code snippets and explanation through the code as well.. they still would have to write/compile the entire code anyway, so that's still a challenge